app/src/docs/components/CanvasModuleHelp/index.jsx

```diff
@@ -17,16 +17,35 @@ type PortHelpProps = {
 // doesn't play well with inline elements.
 const capitalizeText = (text: string): string => text.charAt(0).toUpperCase() + text.slice(1)
 
+const hasTrailingFullStop = (text: string): string => {
+    if (text.slice(-1) === '.') {
+        return true
+    }
+    return false
+}
```

Comment:
```JS
const hasTrailingFullStop = (text: string): boolean => text.trim().endsWith('.')
```

Reply:
even better:

```JS
// append a trailing full stop if `text` does not end with one.
const ensureTrailingFullStop = (text: string): string => text.trim().endsWith(`.`) ? text.trim() : `${text.trim()}.`
// …
ensureTrailingFullStop(capitalizeText(type))
// …
<ReactMarkdown source={ensureTrailingFullStop(capitalizeText(help))} />
```

Reply:
hm, I wonder if it'd actually be better to transform the original text in bulk, as there might be cases where an auto-injected `.` doesn't work e.g. if the help text ends with a link or some other markup 

e.g. 
[link.]()
would become:
[link.]().